README.md

```diff
@@ -1,9 +1,10 @@
-How to use:
-
+## How to use:
 
 - Install product: Eclipse IDE for Eclipse Committers, version: Photon
 - Install VIATRA SDK 2.0.1 http://download.eclipse.org/viatra/updates/release/latest
```

Comment:
The mentioned site contains VIATRA 2.2.0 only; this should either be written as 2.0.1 or newer; or the same version should be specified as Gradle downloads. I know this is not related to the original isue, but we should aim to keep the installed VIATRA version the same as the runtime one to avoid issues with incorrect compiler versions.